imports/server/api/resources/users.ts

```diff
@@ -37,14 +36,12 @@ const ACTIVE_THRESHOLD = 365 * 24 * 60 * 60 * 1000;
 const renderUser = function renderUser(user: Meteor.User, profile: ProfileType) {
   const active = user.lastLogin &&
           Date.now() - user.lastLogin.getTime() < ACTIVE_THRESHOLD;
-  const operator = deprecatedUserMayMakeOperator(user._id);
 
   return {
     _id: user._id,
     primaryEmail: user.emails && user.emails[0].address,
     googleAccount: profile.googleAccount,
     active,
-    operator,
```

Comment:
Confirmed safe to remove at https://github.com/deathandmayhem/jolly-roger/pull/463#discussion_r779212334 (and we know of no other API clients)